code/modules/mob/language/outsider.dm

```diff
@@ -101,19 +101,19 @@
 /datum/language/revenant
 	name = LANGUAGE_REVENANT
 	desc = "The language of the forsaken bluespace inhabitants."
-	speech_verb = list("gargles")
-	ask_verb = list("gags")
-	exclaim_verb = list("retches")
+	speech_verb = list("hisses")
+	ask_verb = list("whispers")
+	exclaim_verb = list("wails")
 	sing_verb = list("trills")
 	colour = "revenant"
 	key = "c"
 	syllables = list("grhhg", "ghrohg", "grgugh", "grrhh", "hghh", "rghghh", "gghhh", "ggrh", "aghrh")
 	flags = RESTRICTED
-	partial_understanding = list(LANGUAGE_TCB = 80)
+	partial_understanding = list(LANGUAGE_TCB = 100)
```

Comment:
is this needed if its 100?

Reply:
This is the easy way to allow revenants to keep their spooky blue voice (since they're speaking a language, really, not actually in Tau Ceti Basic in-game) whilst also being understandable.

I think allowing for flavorful dialogue text in-game (if not different fonts) for situations outside of languages might just cause problems if it's implemented in some other way. I would rather see revenants be the exception to this and not the rule, because later down the line it'd create confusion that someone is speaking a language if someone talking happens to have their font different and also colored.